src/components/views/JobForm.js

```diff
@@ -1,32 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
 import { Field } from 'react-final-form';
 import { FileUploaderField } from '@folio/stripes-erm-components';
 import { AppIcon, TitleManager } from '@folio/stripes/core';
 import stripesFinalForm from '@folio/stripes/final-form';
 
 import {
   Button,
+  Col,
   IconButton,
+  MessageBanner,
   Pane,
   PaneFooter,
   PaneMenu,
   Paneset,
+  Row,
+  TextField,
 } from '@folio/stripes/components';
 
+import { required } from '../../util/validators';
```

Comment:
Can you prefer importing validators from stripes-erm-components where possible? We're slowly trying to cut down on the duplication of code that came about with those so we definitely don't want to _add_ usage of `ui-<module>` validators.